containers/clojure/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+{
+	"name": "Clojure",
```

Comment:
Similar comment here - we're using "(Community)" for community contributed definitions.